google_takeout_parser/parse_json.py

```diff
@@ -23,13 +23,19 @@
 # This is also the 'My Activity' JSON format
 def _parse_json_activity(p: Path) -> Iterator[Activity]:
```

Comment:
wondering if it would make sense to return `Iterator[Res[Activity]]` here (and in similar other functions), and catch/yield exceptions while parsing individual objects here? 
Since currently if there is some field missing, or fails to parse it crashes the whole thing :)

Reply:
Yeah, thats probably a good idea -- especially since this might change in the future, made #10